Tests/Library/Client.tests.ts

```diff
@@ -626,7 +626,7 @@ describe("Library/Client", () => {
             client1.commonProperties = commonproperties;
             client1.config.samplingPercentage = 99;
             mockData.baseData.properties = JSON.parse(JSON.stringify(properties));
-            var env = client1.getEnvelope(mockData);
+            var env = <any>client1.getEnvelope(mockData);
```

Comment:
IIRC it's not casted because of sample rate, but the basedata assertions. The base type in our schema doesn't have the properties object.